ui/src/app/features/user-list/user-list-facade.ts

```diff
@@ -82,7 +82,6 @@ export class UserListFacade implements IFacade {
       deleteUser({
         userId: deletion.userToDelete.userId,
         deleterUserId: deletion.deleterUserId,
-        newOwner,
```

Comment:
please delete newOwner above in params  deleteUser(deletion: UserDeletion, newOwner?: string): void {

Reply:
check where deleteUser is called and delete newOwner there as well if used

Reply:
done